common/src/main/java/com/microsoft/identity/common/internal/request/AdalBrokerRequestAdapter.java

```diff
@@ -198,6 +209,7 @@ public BrokerAcquireTokenSilentOperationParameters brokerSilentParametersFromBun
                 AuthenticationConstants.Broker.ACCOUNT_CORRELATIONID
         );
         if (TextUtils.isEmpty(correlationIdString)) {
+            Logger.info(TAG, "Correlation id not set by Adal, creating a new one");
```

Comment:
verbose

Reply:
should happen very rarely, lets keep it at `info` as it might help with some correlation id issues we were seeing